arch/xtensa/src/common/xtensa_dumpstate.c

```diff
@@ -232,7 +232,11 @@ static inline void xtensa_btdump(void)
   int i;
   bool corrupted = false;
 
-  xtensa_backtrace_start(&pc, &sp, &npc);
```

Comment:
is xtensa_backtrace_start still used anywhere? if not, maybe arch/xtensa/src/common/xtensa_dumpstate.c can be removed?

Reply:
No, it is not used anywhere else. Ok, I will remove it!